File: Modul/ResepMakanan.py
class ResepMakanan():
    def __init__(self, namaResep, bahanUtama, waktuMasak):
        self.__namaResep = namaResep
        self.__bahanUtama = bahanUtama
        self.__waktuMasak = waktuMasak
        
        
    def getNamaResep(self):
        return self.__namaResep
    
    def getBahanUtama(self):
        return self.__bahanUtama
    
    def getWaktuMasak(self):
        return self.__waktuMasak
    
    def __str__(self):
        return f"Nama Resep = {self.__namaResep}, Bahan utama = {self.__bahanUtama}, Waktu = {self.__waktuMasak}"
    

File: Modul/ResepMakanan.java
public class ResepMakanan {
    private String namaResep;
    private String bahanUtama;
    private Integer waktuMemasak;

    public ResepMakanan(String namaResep, String bahanUtama, Integer waktuMemasak) {
        this.namaResep = namaResep;
        this.bahanUtama = bahanUtama;
        this.waktuMemasak = waktuMemasak;
    }

    public String getNamaResep() {
        return namaResep;
    }

    public String getBahanUtama() {
        return bahanUtama;
    }

    public Integer getWaktuMemasak() {
        return waktuMemasak;
    }

    @Override
    public String toString() {
        return "Nama Resep = " + namaResep + ", Bahan Utama = " + bahanUtama + ", Waktu Memasak = " + waktuMemasak;
    }

}
